src/cryptonote_core/service_node_quorum_cop.cpp

```diff
@@ -212,15 +213,15 @@ namespace service_nodes
     service_nodes::quorum_type const max_quorum_type = service_nodes::max_quorum_type_for_hf(hf_version);
     bool tested_myself_once_per_block                = false;
 
-    time_t const now         = time(nullptr);
-    uint64_t const live_time = (now - m_core.get_start_time());
+    time_t const now          = time(nullptr);
+    ptrdiff_t const live_time = (now - m_core.get_start_time());
```

Comment:
It seems like a plain `int` would work fine here (`ptrdiff_t` seems a bit misleading since what's being subtracted here has nothing to do with the size of a pointer).